src/main/resources/publisher_channel_registry_v2.csv

```diff
@@ -3467,4 +3467,9 @@
 "3069FF25-1EBE-411E-B0A3-1EFFC5BA0441";"Norges Handelshøyskole"
 "F8583D5A-FD50-4854-9FEA-F5661AB33E90";"Norsk Utenrikspolitisk Institutt"
 "722A2A28-E078-4ABA-AA5B-6248E5142051";"Nordlandsforskning"
-"11615D7E-8C0C-4748-9F26-784E436F80A3";"IEEE"
\ No newline at end of file
+"11615D7E-8C0C-4748-9F26-784E436F80A3";"IEEE"
+"B2265D97-91CD-4E7C-851F-36D9ACB32DE8";"Norges geologiske undersøkelse"
+"22139870-8d31-4df9-bc45-14eb68287c4a";"RBUP Øst og Sør"
+"8B9EC1FA-3C4A-4875-AA0A-C1E0219C840E";"Norges miljø- og biovitenskapelige universitet"
+"D61B0D47-C78A-48DC-8537-3AD87DEF4D5B";"Norges teknisk- naturvitenskapelige universitet"
+"D61B0D47-C78A-48DC-8537-3AD87DEF4D5B";"NTNU"
```

Comment:
Legger disse i felles publisher fil da det er mappings for ntnu og nmbu hos andre institusjoner